pkg/apis/dashboard/extension.go

```diff
@@ -93,17 +93,17 @@ func (h Handler) CollectionRouteGetBasicInformation(
 	}
 
 	// Count services below owned projects.
```

Comment:
serviceNum->resourceNum

Reply:
Dashboard info includes both serviceCount and resourceCount, see later commits